authors/apps/favorites/tests/base_test.py

```diff
@@ -0,0 +1,68 @@
+from django.urls import reverse
+
+from rest_framework.test import APITestCase, APIClient
+
+
+def generate_slug_url(slug):
+    """Include slug in url path"""
+    full_url = reverse('favorites:user-favorite', args=[slug])
+    return full_url
+
+
+class FavoriteTestBase(APITestCase):
+    """
+    This is the base test class that shall be inherited by favorite test files
+    """
+
+    def setUp(self):
+        """Set up configurations that shall be run every time a test runs"""
+        self.user = {
+            "user": {
+                "username": "tester",
+                "email": "tester123@gmail.com",
+                "password": "tester232#$$"
+            }
+        }
+        self.second_user = {
+            "user": {
+                "email": "tester123@gmail.com",
+                "password": "tester232#$$"}
+        }
+        self.article = {
+            "article": {
+                "title": "The One",
+                "description": "This is an article about the one",
+                "body": "The one is the one",
+                "tagList": ["sample", "article"],
+                "author": 1}
+        }
+        self.article_slug = "the-one"
+        self.non_existent_article_slug = "article-absent"
+        self.client = APIClient()
+        self.register_url = reverse('authentication:register_url')
+        self.login_url = reverse('authentication:login_url')
+        self.articles_url = reverse('articles:articles')
+        # self.favorite_url = reverse('favorites:favorites')
```

Comment:
Remove this commented line

Reply:
@jomasim, I have removed the commented code. thanks again for reviewing

Reply:
@johnwayodi cool otherwise good work I have successfully tested the feature locally.